internal/bundles/manifest.go

```diff
@@ -53,9 +53,14 @@ type Metadata struct {
 	HasParameters   bool            `json:"has_parameters,omitempty"`   // True if this is content allows parameter customization.
 }
 
+type EnvironmentPython struct {
+	PythonRequires string `json:"requires"` // The Python version to use for the content environment
+}
+
 type Environment struct {
-	Image    string `json:"image"`    // The image to use during content build/execution
-	Prebuilt bool   `json:"prebuilt"` // Determines whether Connect should skip the build phase for this content.
+	Image    string             `json:"image"`            // The image to use during content build/execution
+	Prebuilt bool               `json:"prebuilt"`         // Determines whether Connect should skip the build phase for this content.
+	Python   *EnvironmentPython `json:"python,omitempty"` // If non-null, specifies the Python environment
```

Comment:
I didn't catch this before when Brian was going over the feature. Why didn't y'all place this under the python section in the manifest?